Source/Libunwind Regression Tests/unwind_test_harness.c

```diff
@@ -128,7 +128,10 @@ static struct unwind_test_case unwind_test_cases[] = {
     /* frame-based unwinding */
     { unwind_tester_list_x86_64_frame,      false,  frame_readers_frame,    2 },
     { unwind_tester_list_x86_64_frame,      true,   frame_readers_compact,  2 },
+#if !TARGET_OS_SIMULATOR
+    /* FIXME: this doesn't work on iOS and tvOS simulators - failed to find DWARF's FDE section. */
```

Comment:
Is it really fixable?

Reply:
Well, it doesn't seem so. But it's unclear why there is no data for it. It would be nice if some asm expert have a look at it

Reply:
I'll create a bug we'll see if we can address this separately.